src/reducers/search.js

```diff
@@ -3,7 +3,7 @@ import moment from 'moment';
 import { createReducer } from 'redux-action-tools';
 import {
   SEARCH_ARTICLES, UPDATE_SEARCH_QUERY, CHANGE_COLUMN_VISIBILITY,
-  SET_SEARCH_CONDITION, ADD_DATE_RANGE, REMOVE_DATE_RANGE, RESET_DATE_RANGE,
+  SET_SEARCH_CONDITION, ADD_DATE_RANGE, REMOVE_DATE_RANGE, RESET_DATE_RANGE, CHANGE_SORT_METHOD,
```

Comment:
If the code is too long in a single line (over 80 words per line), you can put some constants at a new line.

Reply:
Finished